stl/inc/sstream

```diff
@@ -519,7 +524,10 @@ public:
     using _Mysb          = basic_stringbuf<_Elem, _Traits, _Alloc>;
     using _Mystr         = basic_string<_Elem, _Traits, _Alloc>;
 
-    explicit basic_stringstream(ios_base::openmode _Mode = ios_base::in | ios_base::out)
+

```

Comment:
There's an extra newline being introduced here.

Reply:
Removed the new line